### DISCUSSION TITLE
Skip menu if i selected a html tag

### DISCUSSION THREAD

**Discussion by AlejandroSanchez90** [saghen/blink.cmp#659] [General]
i would like to know how to disable menu if i insert a tag, right now i write div, to then get the completion for <div></div>, my cursor will end up inside the tag, but before blink i would press enter and it will indent properly, but now after i select div, the menu stays open and using Enter will just keep selecting the first item in the menu (my confirmation key is enter)

**Comment by saghen** [comment]
What's the output of `:lua vim.print(require('blink.cmp.sources.lib').get_trigger_characters('default'))` ?

**Comment by AlejandroSanchez90** [comment]
that was fast
{ "/", ".", "\t", "\n", ".", ":", "(", "'", '"', "[", ",", "#", "*", "@", "|", "=", "-", "{", " ", "+", "?" }

the menu is always open, if I have a variable called test, I write test, I see the completion, I hit enter, and the menu reopens, even though I have not typed a '.' so it knows that I want a second arg there, i can spam enter and the menu will open again and again
